wallet/src/wallet/tests.rs

```diff
@@ -681,7 +681,7 @@ fn issue_and_transfer_tokens(#[case] seed: Seed) {
 
     let amount_fraction = (block1_amount.into_atoms() - NETWORK_FEE) / 10;
 
-    let token_amount_to_issue = Amount::from_atoms(rng.gen_range(1..amount_fraction));
+    let token_amount_to_issue = Amount::from_atoms(rng.gen_range(2..amount_fraction));
```

Comment:
> Looks like we need to issue a minimum of 2 tokens (was min 1 before) since the test expects to transfer some and ensure there's still an existing balance (min 1) left.

Not critical but would be nice if this was mentioned in a comment here.